autoware_launch/launch/autoware.launch.xml

```diff
@@ -121,16 +119,12 @@
   </group>
 
   <!-- API -->
-  <include file="$(find-pkg-share tier4_autoware_api_launch)/launch/autoware_api.launch.xml">
-    <arg name="init_simulator_pose" value="$(var init_simulator_pose)"/>
-    <arg name="init_localization_pose" value="$(var launch_localization)"/>
-  </include>
+  <include file="$(find-pkg-share tier4_autoware_api_launch)/launch/autoware_api.launch.xml"/>
```

Comment:
```suggestion
  <group>
  <include file="$(find-pkg-share tier4_autoware_api_launch)/launch/autoware_api.launch.xml"/>
  </group>
```
I heard that include tags should be wrapped in group tags.

Reply:
https://github.com/autowarefoundation/autoware.universe/issues/1160

Reply:
Done https://github.com/autowarefoundation/autoware_launch/pull/72/commits/237713af76a9e3d133f027e4be7183ed81dfd144.